tests/test_routing.py

```diff
@@ -1147,3 +1147,52 @@ def test_distance_to_channel_mfd_with_weights(self):
 
         numpy.testing.assert_almost_equal(
             distance_to_channel_d8_array, zero_array)
+
+    def test_flow_dir_mfd_plateau(self):
+        """PGP.routing: MFD on a plateau."""
+        import pygeoprocessing.routing
+
+        driver = gdal.GetDriverByName('GTiff')
+        workspace_dir = 'flow_dir_debug'

```

Comment:
Should this be `workspace_dir = os.path.join(self.workspace_dir, 'flow_dir_debug') ` or something?

Reply:
Even better, it should be removed. I do that a lot when I'm doing a single test and want to inspect the results, then i unintentionally leave it there. removed in 0171f2d5c540901f5f144bc37deca44957244f13